CONTRIBUTING.md

```diff
@@ -1,59 +1,91 @@
+## Create a pull request
+
+- Try to keep the pull requests small. A pull request should try its very best to address only a single concern.
+- For work in progress pull requests, please use the Draft PR feature.
+- Make sure all tests pass and add additional tests for the code you submit.
+- Document your reasoning behind the changes. Explain why you wrote the code in the way you did. The code should explain what it does.
+- If there's an existing issue, reference to it by adding something like `References/Closes/Fixes/Resolves #123`, where 123 is the issue number. 
+- Please fill out the PR Template when making a PR.
+
+> Please note: maintainers may close your PR if it has gone stale or if we don't plan to merge the code.
 
-## Contributors
-### Financial Contributors
+## Pull request reviews
 
-Becoming a [financial contributor](https://opencollective.com/CircuitVerse/contribute) helps us sustain our community and platform.
+- Requested changes must be resolved (with code or discussion) before merging.
+- If you make changes to a PR, be sure to re-request a review.
+- Don't repeadetely tag someone(may be it is not the right time to review your PR), be patient.
+- Do not 'resolve conversation' unnecessary raised by a community member or any workflow tools(codeclimate or hound) as they may have some purpose, try to resolve the request changes and if any help wanted tag a communtiy member to give views about that.
 
-#### Individuals
-<a href="https://opencollective.com/CircuitVerse"><img src="https://opencollective.com/CircuitVerse/individuals.svg?width=890" alt="Individual financial contributors"></a>
+#### PR Lebels
 
-#### Organizations
-Support this project with your organization. Your logo will show up here with a link to your website. [[Contribute](https://opencollective.com/CircuitVerse/contribute)]
+- `under-review` labeled PRs are under review by core team.
+- `waiting for contributor` labeled PRs are meant to waiting for contributor to respond.
+- `waiting for design` labeled PRs are meant to waiting for review from UI/UX core team.
+- `no activity` labeled PRs are meant to have no activity in the PR from since a while.
+- `blocked` labeled PRs are meant not to go ahead for review.
+- `do not merge` labeled PRs are meant not to merge the PR right now(may be later).
+- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PRs are meant to be waiting for other communtiy members.
 
-<a href="https://opencollective.com/CircuitVerse/organization/0/website"><img src="https://opencollective.com/CircuitVerse/organization/0/avatar.svg" alt="Organization 1"></a>
```

Comment:
It seems that financial contrinution section via open collective is removed, Can you add them back ?

Reply:
Oh sorry, I forget to add it.
Added